apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -161,6 +162,15 @@ defmodule Aecore.Chain.Worker do
     Logger.info(fn ->
       "Added block ##{new_block.header.height} with hash #{Base.encode16(new_block_hash)}, total tokens: #{inspect(total_tokens)}"
     end)
+
+    # Broadcasting notifications for new mined transaction(per account)
+    for tx <- new_block.txs do
```

Comment:
for returns a list of the return values in each iteration, Enum.each is more suitable here, it will just return :ok